packages/react-native/android/src/main/java/com/bugsnag/reactnative/MapExtensions.kt

```diff
@@ -13,6 +13,7 @@ internal fun Map<String, Any?>.toWritableMap(): WritableMap {
             null -> nativeMap.putNull(key)
             is Boolean -> nativeMap.putBoolean(key, obj)
             is Int -> nativeMap.putInt(key, obj)
+            is Long -> nativeMap.putDouble(key, obj.toDouble())
```

Comment:
Is `toDouble()` the same as `as Double` used below?
```suggestion
            is Long -> nativeMap.putDouble(key, obj as Double)
```

Reply:
Updated to use `.toDouble()` for both, rather than relying on casts.